tox.ini

```diff
@@ -1,18 +1,18 @@
 [tox]
-envlist = py{34,35,36}-{core,testtools},docs,pep8
+envlist = py{34,35,36}-{core,testtools,full},docs,pep8
 
 [testenv]
 extras =
-    test
     testtools: testtools
+    full: test
 deps =
-    coverage
+    full: coverage
 commands =
     pip list
-    # Assume core tests will always run first and have them erase coverage.
-    core: coverage run -m pytest {posargs:seaworthy/tests}
-    testtools: coverage run -a -m pytest {posargs:seaworthy/tests-testtools}
-    coverage report -m
+    core: {envbindir}/python -m unittest discover {posargs:seaworthy/tests-core}
```

Comment:
Can you explain this syntax? `{posargs:seaworthy/tests-core}`.. is that defaulting to `seaworthy/tests-core`?